util/coordinator.py

```diff
@@ -415,15 +417,15 @@ def start(self):
         if self.is_chief:
             log_debug('Starting coordinator...')
             self._thread = Thread(target=self._httpd.serve_forever)
-            self._thread.daemon = True
+            # self._thread.daemon = True
```

Comment:
Out of curiosity, why did this change?

Reply:
Leftover from debugging, will remove this.